ci-operator/step-registry/assisted/common/setup/post/assisted-common-setup-post-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ assisted common setup post command ************"
+
+mkdir -p build/ansible
+cd build/ansible
+
+# Get packet | vsphere configuration
+# shellcheck source=/dev/null
+set +e
+source "${SHARED_DIR}/packet-conf.sh"
+source "$SHARED_DIR/ci-machine-config.sh"
+set -e
+
+# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
+~/fix_uid.sh
+
+cat << EOF > inventory
+[all]
+${IP} ansible_user=root ansible_ssh_user=root ansible_ssh_private_key_file=${SSH_KEY_FILE} ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90 -o LogLevel=ERROR"
+EOF
+
+cat > run_post_playbook.yaml <<-EOF
+- name: Prepare remote host
```

Comment:
this reoccurring in some places, and should be more indicative

Reply:
not sure that i understand what you meant. I don't like creating files as part of the script also but current there is no other chioce.
Adding those file to a separated repo might be unsafe since currently the CI tests every change.

Reply:
we're documenting all ansible playbooks as ``name: Prepare remote host``, and it needs to be more specific to the actual action
in this case:
```
- name: Run post-install commands
```